runbot_travis2docker/models/runbot_branch.py

```diff
@@ -5,12 +5,47 @@
 
 import re
 
-import requests
 import subprocess
+import requests
 
 from openerp import fields, models, api
 
 
+def read_group(branch):
```

Comment:
@moylop260 
Thanks for teach me the annotation `@tools.ormcache` it is very powerfull